### ISSUE TITLE
Build error on Fedora 40

### ISSUE THREAD

**Issue by ycollet** [grame-cncm/faust#1008]
I am building Faust 2.72.13 on Fedora 40 with gcc 14.0.1 and I met an error during build. A header is missing:
```
diff --git a/compiler/generator/llvm/llvm_dynamic_dsp_aux.cpp b/compiler/generator/llvm/llvm_dynamic_dsp_aux.cpp
index 1643097..d3f17a8 100644
--- a/compiler/generator/llvm/llvm_dynamic_dsp_aux.cpp
+++ b/compiler/generator/llvm/llvm_dynamic_dsp_aux.cpp
@@ -43,6 +43,7 @@
 #include <llvm/IR/LegacyPassNameParser.h>
 #include <llvm/IR/Verifier.h>
 #include <llvm/IR/PassManager.h>
+#include <llvm/IR/Constants.h>
 #include <llvm/IRReader/IRReader.h>
 #include <llvm/Linker/Linker.h>
 #include <llvm/Support/FileSystem.h>
-- 
2.44.0
```

**Comment by sletz** [comment]
Thanks, will be fixed in next 2.72.14.

**Comment by ycollet** [comment]
Thanks a lot !
